### PR TITLE
[SPH-TASK] [FE] Create Welcome Back Page

### PR DESCRIPTION
**Pull request by misterRenz**
## Task Link
- https://app.asana.com/0/1202718774787420/1202733801001927/f

## Commands
- npm start

## Pre-conditions
- The Frontend developer must be working in the frontend folder of the repository

## Expected Output
- It will display the Welcome Back Page!
- It will display a button that states "Continue"
- Button will not work for now because it is still not linked to dashboard page because it is still not implemented.

## Notes
-Continue button is still not linked to dashboard page.

## Screenshots
image

### CODE REVIEW

File: frontend/src/Components/WelcomePage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
```
Comment (karlolee50) [inline frontend/src/Components/WelcomePage.js]: the Components folder should be lowercase. please rename it to `components`

File: frontend/src/Components/WelcomePage.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          <div>
+            <button className="btn-continue">Continue</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default WelcomePage;
```
Comment (karlolee50) [inline frontend/src/Components/WelcomePage.js]: please also move the WelcomePage to a sub folder -> `frontend/src/components/welcome-page/index.js`

**Review by karlolee50 — CHANGES_REQUESTED** [review]
please check comments

File: frontend/src/Components/WelcomePage.js (hunk 2 above)
Comment (misterRenz) [reply frontend/src/Components/WelcomePage.js:24]: So all welcome page related files will be on one folder which also includes its css?
image

File: frontend/src/components/welcome-page/WelcomePage.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
+h1 {
```
Comment (karlolee50) [inline frontend/src/components/welcome-page/WelcomePage.css]: you can rename this to index.css as well

File: frontend/src/components/welcome-page/index.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./WelcomePage.css";
+
+const WelcomePage = () => {
+  return (
+    <div className="container">
+      <h1>Home Inventory System</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```
Comment (karlolee50) [inline frontend/src/components/welcome-page/index.js]: maybe you can remove this if not needed. this is just filler text right? or do we already have text that should be placed here?

**Review by karlolee50 — CHANGES_REQUESTED** [review]
please check

**Review by karlolee50 — APPROVED** [review]
approved

**Pull request merged**